docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
+The configurations you set in the catalog configuration file will apply to all the catalogs of the same type that you created.
+For detailed configurations about each catalog, see related catalog documentation.
 
 :::note
-Gravitino takes the catalog configurations in the following order:
+Gravitino takes the catalog configurations in the following order of precedence:
 
-1. Catalog `properties` specified in catalog creation API or REST API.
+1. Catalog `properties` specified in the catalog creation API or REST API.
 2. Catalog configurations specified in the catalog configuration file.
 
-The catalog `properties` can override the catalog configurations specified in the configurationfile.
+The catalog `properties` overrides the catalog configurations specified in the configuration file.
 :::
 
-Gravitino supports passing in catalog-specific configurations if you add `gravitino.bypass.`. For example, if you want to pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog, add the `gravitino.bypass.` prefix.
+Gravitino supports passing in catalog-specific configurations by adding the `gravitino.bypass.` prefix to the specific catalog configurations.
+For example, you can pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog if you add the `gravitino.bypass.` prefix to the configuration.
 
-Also, Gravitino supports loading catalog-specific configurations from external files. For example,you can put your own `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino loads it automatically.
+Gravitino also supports loading catalog-specific configurations from external files.
+You just need to put your `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino will load it automatically.
 
-#### Start Apache Gravitino server
+### Start the Gravitino server
 
-After configuring the Gravitino server, start the Gravitino server by running:
+- Start the Gravitino server
 
-```shell
-./bin/gravitino.sh start
-```
+  After configuring the Gravitino server, run the following command to start the Gravitino server:
 
-Alternatively, to start the Gravitino server Web UI, please run:
+   ```shell
+   ./bin/gravitino.sh start
+   ```
 
-```shell
-./bin/gravitino.sh run
-```
+- Start the Gravitino server with the Web UI
 
-You can access the Gravitino Web UI by typing [http://localhost:8090](http://localhost:8090) in your browser, or you can run:
+  By default, the Gravitino server also provides a Web UI on port `8090`.
+  Run the following command to start the Gravitino server with the Web UI:
+
+   ```shell
+   ./bin/gravitino.sh run
+   ```
+
+After starting the Gravitino server with the Web UI, visit `http://localhost:8090` in your browser to access the Gravitino server through the Web UI.
+Or, you can run the following command to verify that the Gravitino server is running:
```

Comment:
verify -> check?

Reply:
I think both words are ok.

Reply:
Check is less formal, and verify implies more work. While both are OK, it seems a little odd to me as a native English speaker to use verify in this context.